test/libsolidity/syntaxTests/abiEncoder/abi_encodeCall_tuple_returned_from_function.sol

```diff
@@ -0,0 +1,26 @@
+contract C {
+    function g0() internal {}
+    function g1() internal returns (uint) { return (1); }
+    function g2() internal returns (uint, uint) { return (2, 3); }
+
+    function f0() public {}
+    function f1(uint) public {}
+    function f2(uint, uint) public {}
+
+    function h() public view {
+        abi.encodeCall(this.f0, g0());
+        abi.encodeCall(this.f1, g1());
```

Comment:
```suggestion
        abi.encodeCall(this.f1, g1()); // OK
```

Reply:
Actually, maybe we even should have a test covering this case with all `abi.` functions. I'm not sure we already have one.

Reply:
I created separated tests for each `abi.encode` function.